server/controllers/items/lib/items.coffee

```diff
@@ -66,11 +66,15 @@ module.exports = items_ =
     db.viewByKeys 'byOwnerAndEntityAndListing', keys
     .map filterPrivateAttributes(reqUserId)
 
-  create: (userId, item)->
-    db.post Item.create(userId, item)
+  create: (userId, items)->
+    _.type items, 'array'
+    Promise.all items.map(snapshotEntityData)
```

Comment:
it would be a great opportunity to describe/document what `snapshotEntityData` is doing, cuz its a bit cryptic and its fundamental to understand the review  :)
maybe at the beginning of the file  `items/lib/snapshot/snapshot_entity_data.coffee` directly ?

Reply:
yes! done in 55d6127